bigquery/google/cloud/bigquery/table.py

```diff
@@ -713,6 +713,137 @@ def _build_resource(self, filter_fields):
         return resource
 
 
+class TableListItem(object):
+    """Read-only table resource object with a subset of table properties.
```

Comment:
@tswast Can you add another sentence or two before "See..." explaining why this type exists and where the data comes from?

Reply:
Added.